.rubocop_todo.yml

```diff
@@ -24,11 +24,65 @@ Gemspec/RequiredRubyVersion:
     - 'engines/order_management/order_management.gemspec'
     - 'engines/web/web.gemspec'
 
+# Offense count: 28
```

Comment:
I'm surprised to see so many new exclusions added to the todo file. I guess this is code that was added before these rules were enabled, and went unnoticed because I think we only lint added code. 
So this is simply  documenting all the exceptions that we didn't know we had.